src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/change_point/fields_suggestions_after.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import { type ESQLAstCommand, type ESQLAstChangePointCommand, LeafPrinter } from '@kbn/esql-ast';
+import type { ESQLRealField } from '../../../validation/types';
+
+export const fieldsSuggestionsAfter = (
+  command: ESQLAstCommand,
+  previousCommandFields: ESQLRealField[],
+  userDefinedColumns: ESQLRealField[]
+) => {
+  const { target } = command as ESQLAstChangePointCommand;
+  previousCommandFields.push(
+    {
+      name: target ? LeafPrinter.column(target.type) : 'type',
```

Comment:
What is the `LeafPrinter` thing? This seems to be doing basically the same thing as the [`getSafeInsertText`](https://github.com/elastic/kibana/blob/7dbe44a91732f35bb46467d969618612d07cf075/src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/factories.ts#L60-L64)

Does it make sense to dedupe?

Reply:
I followed Vadim's comment, this looks fine to me. We can use the Leafprinter instead of getSafeInsert in other places too 👍